plugin/api.go

```diff
@@ -254,6 +254,11 @@ type API interface {
 	// Minimum server version: 5.6
 	GetProfileImage(userId string) ([]byte, *model.AppError)
 
+	// GetEmojiList returns a page of custom emoji on the system.
+	//
+	// Minimum server version: 5.6
+	GetEmojiList(sortBy string, page, perPage int) ([]*model.Emoji, *model.AppError)
```

Comment:
I know I'm being very nit picky, but I propose to set `sortBy` as the last parameter to keep it consistent with the other API methods.

Reply:
@hanzei, do you mean relative to the App? It seems like the plugin API always puts `sortBy` first, e.g. https://github.com/mattermost/mattermost-server/blob/06b124d2ff188817431fbd64fc7aaeb0075d15c2/app/plugin_api.go#L204-L213

Reply:
Sorry, I meant relative to the App. In the plugin API there is currently just one `sortBy` parameter in https://developers.mattermost.com/extend/plugins/server/reference/#API.GetUsersInChannel. But since this change is consistent with the existing API method, never mind my comment.